### ISSUE TITLE
What can I do to use the dice metric in this code?

### ISSUE THREAD

**Issue by li-pengcheng** [zizhaozhang/unet-tensorflow-keras#14]
at line136 in train.py, i add:  dice = dice_coef_loss(y_batch[0], pred_map) but it seems don't work, could you please give me some information about how to use this metric, thx!
